src/H5win32defs.h

```diff
@@ -68,26 +70,38 @@ struct timezone {
 #define HDopen(S, F, ...) Wopen_utf8(S, F, ##__VA_ARGS__)
 #endif
 
-#define HDread(F, M, Z)       _read(F, M, Z)
-#define HDremove(S)           Wremove_utf8(S)
-#define HDrmdir(S)            _rmdir(S)
-#define HDsetenv(N, V, O)     Wsetenv(N, V, O)
-#define HDsetvbuf(F, S, M, Z) setvbuf(F, S, M, (Z > 1 ? Z : 2))
-#define HDsleep(S)            Sleep(S * 1000)
-#define HDstat(S, B)          _stati64(S, B)
-#define HDstrcasecmp(A, B)    _stricmp(A, B)
-#define HDstrdup(S)           _strdup(S)
-#define HDstrtok_r(X, Y, Z)   strtok_s(X, Y, Z)
-#define HDtzset()             _tzset()
-#define HDunlink(S)           _unlink(S)
-#define HDunsetenv(N)         Wsetenv(N, "", 1)
-#define HDwrite(F, M, Z)      _write(F, M, Z)
+#define HDread(F, M, Z)     _read(F, M, Z)
+#define remove(S)           Wremove_utf8(S)
+#define HDrmdir(S)          _rmdir(S)
+#define HDsetenv(N, V, O)   Wsetenv(N, V, O)
+#define sleep(S)            Sleep(S * 1000)
+#define HDstat(S, B)        _stati64(S, B)
+#define HDstrdup(S)         _strdup(S)
+#define HDstrtok_r(X, Y, Z) strtok_s(X, Y, Z)
+#define HDtzset()           _tzset()
+#define HDunlink(S)         _unlink(S)
+#define HDunsetenv(N)       Wsetenv(N, "", 1)
+#define HDwrite(F, M, Z)    _write(F, M, Z)
```

Comment:
What's your plan for dealing with this? Removing HD from the C calls is great and all, but leaving a handful of calls that require the HD prefix is worse than leaving them all in place.